openprescribing/frontend/models.py

```diff
@@ -349,12 +349,9 @@ class Presentation(models.Model):
                                 validators=[isAlphaNumeric])
     name = models.CharField(max_length=200)
     is_generic = models.NullBooleanField(default=None)
-    active_quantity = models.FloatField(null=True, blank=True)
-    adq = models.FloatField(null=True, blank=True)
-    adq_unit = models.CharField(max_length=10, null=True, blank=True)
     is_current = models.BooleanField(default=True)
-    percent_of_adq = models.FloatField(null=True, blank=True)
     replaced_by = models.ForeignKey('self', null=True, blank=True)
+    adq_per_quantity = models.FloatField(null=True, blank=True)
```

Comment:
Please could you add a comment explaining how to think about `adq_per_quantity`?